### PR TITLE
Doc about our Sponsors

### PR DESCRIPTION
**Pull request by zaro0508**
We have Sponsors that provide vital services to the Sceptre
community and so we need to track that info somwhere in case
one of those services goes sideways or needs to be updated.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -203,10 +203,17 @@ $ behave integration-tests --include <feature-file>
 $ behave integration-tests -n "<scenario-name>"
 ```
 
+# Sponsors
+
+* [Sage Bionetworks](https://sagebionetworks.org/) donated the AWS account for running Sceptre integration
+  tests.  Please contact it@sagebase.org for support.
+* [Go Daddy](https://www.godaddy.com/) donated [the domain](https://docs.sceptre-project.org) for hosting the Sceptre project.  Please contact oss@godaddy.com for support.
```
Comment (tarkatronic) [inline CONTRIBUTING.md:210]: GoDaddy should be a single word

File: README.md [hunk 2]
```diff
@@ -266,10 +266,20 @@ Full API reference documentation can be found in the
...
 ## Contributing
 
 See our [Contributing Guide](CONTRIBUTING.md)
+
+
+## Sponsors
+
+[![GoDaddy](sponsors/godaddy_logo.png "GoDaddy")](https://www.godaddy.com)
+
+[![Cloudreach](sponsors/cloudreach_logo.png "Cloudreach")](https://www.cloudreach.com)
```
Comment (jfalkenstein) [inline README.md:283]: Do they still sponsor anything? I thought they dropped all involvement...

Comment (zaro0508) [reply README.md:285]: Yes, they have however the docker for sceptre is deployed to their dockerhub account.  Once it's move to the sceptreorg account then i believe we can drop cloudwatch.   More info in issue #1098

File: CONTRIBUTING.md (hunk 1 above)
Comment (zaro0508) [reply CONTRIBUTING.md]: fixed

**Review by tarkatronic — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (tarkatronic) [reply README.md:285]: I would recommend at least keeping a historical note about Cloudreach's involvement/sponsorship in the long term.

Comment (zaro0508) [reply README.md:285]: done

**Pull request merged**